thirdparty/GTest/CMakeLists.txt

```diff
@@ -7,6 +7,8 @@ if(BUILD_SHARED_LIBS)
 endif()
 set(BUILD_SHARED_LIBS OFF)
 
+set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreadedDebug")
+
```

Comment:
```suggestion
```

This must be set at the `thirdparty/ffs/ffs/scripts/ci/cmake/windows-common.cmake` level